cmd/osv-scanner/main.go

```diff
@@ -155,6 +159,17 @@ func run(args []string, stdout, stderr io.Writer) int {
 				}
 			}
 
+			if context.Bool("experimental-licenses-summary") && context.IsSet("experimental-licenses") {
+				return fmt.Errorf("both --experimental-licenses-summary and --experimental-licenses flags set")
```

Comment:
to me this is pointing out a "fact" rather than a "problem" currently:

```suggestion
				return fmt.Errorf("--experimental-licenses-summary and --experimental-licenses flags cannot be used together")
```